src/StarterKitServiceProvider.php

```diff
@@ -11,28 +11,53 @@
 
 class StarterKitServiceProvider extends PackageServiceProvider
 {
+    const PACKAGE_NAME = 'starterkit';
+
+    const THEME_NAME = 'cwd-framework';
+
     public const INSTALL_FILES = [
         'README.md',
         '.lando.yml',
     ];
 
+    public const ASSET_FILES = [
+        'css',
+        'fonts',
+        'images',
+        'js',
+        'favicon.ico',
+    ];
```

Comment:
@inaydich I think your critique of not including the sass directory was on target. I've limited the published assets to only those used by the layout.